docker-compose.yaml

```diff
@@ -0,0 +1,14 @@
+version: "3.9"
+services:
+  db:
+    image: postgres:latest
+    environment:
+      POSTGRES_USER: postgres
+      POSTGRES_PASSWORD: pw
+      POSTGRES_DB: db
+    ports:
+      - "5432:5432"
+    volumes:
+      - postgres_data:/var/lib/postgresql/data
```

Comment:
just a query here, now that the database lives on the same host as the app, are we sure that if the host goes down we won't lose the data? Not sure exactly how the infrastructure works, but I assume the host is a node with persistent volume provisioned from a k8s cluster or something similar?

Reply:
docker-compose file here spins up a local testing db. The remote prod db is already running in ps5 cluster.